frontend/src/metabase/components/Timeline/Timeline.styled.tsx

```diff
@@ -19,17 +22,17 @@ export const TimelineItem = styled.li`
   margin-bottom: 1.5rem;
 `;
 
-export const ItemIcon = styled(Icon)`
+export const StyledIcon = styled(Icon)`
   position: relative;
   color: ${props => (props.color ? color(props.color) : color("text-light"))};
```

Comment:
If all we're really doing is changing the color, the `Icon` component accepts color as a prop. We may not need a styled component here